<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TubeWidget</class>
 <widget class="QWidget" name="TubeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TubeWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="8,2">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="openXRayBtn">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>ON</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeXRayBtn">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>OFF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="connectBtn">
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="connectComboBox">
         <property name="minimumSize">
          <size>
           <width>61</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>61</width>
           <height>35</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>23</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Voltage：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="voltageSpinBox">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>kV</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="voltageEdit">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>0.00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>kV</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>41</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>41</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Mode：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="modeComboBox">
         <property name="minimumSize">
          <size>
           <width>61</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>61</width>
           <height>35</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>large</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>medium</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>small</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Current：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="currentSpinBox">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="minimumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>uA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="currentEdit">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>0.00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="minimumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>uA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="queryBtn">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Query</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="modeSetBtn">
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>71</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Mode Set</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="statusEdit"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
